cves/CVE-2018-1999015.yml

```diff
@@ -357,11 +393,11 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: CWE-125
+CWE_note: The system could read out of bounds of the file memory.
 nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it. If the
   report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
 
   Must be under 30 characters. Optional. Be appropriate.
-nickname:
+nickname: Unchecked Media Size Issue
```

Comment:
There's nothing wrong with this, but we like to see more catchy/fun nicknames, rather than just simple descriptions.